source/initial_temperature/adiabatic.cc

```diff
@@ -33,210 +33,279 @@ namespace aspect
 {
   namespace InitialTemperature
   {
+
+    template <int dim>
+    Adiabatic<dim>::Adiabatic ()
+      :
+      surface_boundary_id(numbers::invalid_unsigned_int)
+    {}
+
+    template <int dim>
+    void
+    Adiabatic<dim>::initialize ()
+    {
+      // Find the boundary indicator that represents the surface
+      surface_boundary_id = this->get_geometry_model().translate_symbolic_boundary_name_to_id("outer");
+      std::set<types::boundary_id> surface_boundary_set;
+      surface_boundary_set.insert(surface_boundary_id);
+
+      // The input ascii table contains one data column (LAB depths(m)) in addition to the coordinate columns.
+      Utilities::AsciiDataBoundary<dim>::initialize(surface_boundary_set,
+                                                    1);
+    }
+
     template <int dim>
     double
     Adiabatic<dim>::
     initial_temperature (const Point<dim> &position) const
     {
+      double age_top = 0;
+      double age_bottom = (this->convert_output_to_years() ? age_bottom_boundary_layer * year_in_seconds
+                           : age_bottom_boundary_layer);
+      if (read_from_ascii_file)
+        {
+          age_top = Utilities::AsciiDataBoundary<dim>::get_data_component(surface_boundary_id,
+                                                                                position,
```

Comment:
Remove white spaces to align this line and the following one to match the start of `surface_boundary_id`